src/main/java/com/endava/exercise/BeanConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package com.endava.exercise;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class BeanConfiguration {
+
+    @Bean("anja")
```

Comment:
Do you lose any flexibility naming this "anja" and not "pilot"?

Reply:
None that I can think of :-). "anja" is a concrete Pilot.